### PR TITLE
docs(812): add accessibility statement to docs site

### LINKED ISSUE #812: Add Accessibility Statement to NewsKit docs site

### PR DESCRIPTION
**Pull request by JohnTParsons**
closes #812

**What**

1. Background - accessibility awareness
2. What did you do - added statement
3. What does the reviewers should expect - new footer link and page

**I have done:**
- [ ] Written unit tests against changes
- [x] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

File: cypress/site/functional/footer-navigation.cy.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// /<reference types="Cypress" />
+
+describe('Documentation Site - footer-navigation component', () => {
+  describe('Desktop view', () => {
+    it('pages should have footer links', () => {
+      cy.mockConsentAndVisit('/');
+      cy.contains('a', 'Careers').should('exist');
```
Comment (jps) [inline cypress/site/functional/footer-navigation.cy.js:7]: Is this a good e2e test? I'm not so sure, I agree that these won't change that frequently and it's possibly not much of an issue but what feels off to me is that it's very tightly coupled to the content, conceptually to me, this is very close to a snapshot test 🤔

Comment (JohnTParsons) [reply cypress/site/functional/footer-navigation.cy.js:7]: Should I remove footer-navigation.cy.js?
Should I remove the new tests in header-navigation.cy.js too?

Comment (JohnTParsons) [reply cypress/site/functional/footer-navigation.cy.js]: Change pushed to remove footer-navigation.cy.js

**Review by LukeFinch — DISMISSED** [review]

**Review by Vanals — DISMISSED** [review]

File: cypress/site/functional/footer-navigation.cy.js (hunk 1 above)
Comment (JohnTParsons) [reply cypress/site/functional/footer-navigation.cy.js]: @mutebg As requested, new header cypress tests removed as well

**Review by mutebg — APPROVED** [review]

**Pull request merged**